src/main/java/uk/gov/laa/ccms/caab/service/ApplicationService.java

```diff
@@ -233,6 +234,13 @@ public List<BaseApplication> getCases(
     return searchResults;
   }
 
+  public Mono<Void> patchApplication(
```

Comment:
I know it's just a one-liner, but could do with a javadoc.

Reply:
Amended